code/modules/mob/living/simple_animal/eldritch_demons.dm

```diff
@@ -190,6 +190,13 @@
 	adjustBruteLoss(-maxHealth * 0.5, FALSE)
 	adjustFireLoss(-maxHealth * 0.5 ,FALSE)
 
+
+/mob/living/simple_animal/hostile/eldritch/armsy/Shoot(atom/targeted_atom)
+	. = ..()
+	target = targeted_atom
+	AttackingTarget()
```

Comment:
have you tested if this works like we think it will?

Reply:
aaaaaaa fixed